index.js

```diff
@@ -422,18 +419,21 @@ document.addEventListener("DOMContentLoaded", () =>{
 	// Hunter
 
 	
-	// Validation
-	function validator(){
-		nameChar = document.querySelector("#characterName").value;
-		if (nameChar == ""){
-			alert("Name must be filled out");
-			return false;
-		}else{
-			nameWritten = true;
-			save();
-			updater();
-			return false;
+	/**
+	 * Checks if the character name exists in the HTML.
+	 * If yes, saves it in the Character, saves, and updates.
+	 */
+	function nameValidator(){
+		const characterName = document.querySelector('#characterName').value;
+
+		if (!characterName) {
+			return alert('Name must be filled out');
 		}
+
+		nameWritten = true;
+		nameChar = characterName;
+		save();
+		updater();
 	}	
```

Comment:
I've done a couple of things here:

1. Renamed the function - function names should be very explicit in what they do
1. Added some JSDocs comments to the function with the /** * */ - this is the standard for documenting functions
1. Created a temporary variable `characterName` to hold the name, you should do this to validate it before assigning it its real place
1. In JS, functions aren't required to `return` anything, and you can also return it as I've done with some code in front. That line will still be executed.
1. In JS, you can do something like `!characterName`, which checks if it's either null, undefined, false, 0, or an empty string. In this case, it's an emptry string.